src/platform/forms-system/test/js/utilities/file.unit.spec.js

```diff
@@ -25,8 +25,15 @@ const encryptedMockFile = [
 describe('readAndCheckFile', () => {
   let oldFileReader;
 
-  const setup = async (ext = 'pdf', isEncrypted) => {
+  before(() => {
     oldFileReader = global.FileReader;
```

Comment:
This line was getting run every time `setup` is called, so the second and subsequent times, `global.FileReader` was getting overwritten with the mock.